genesis/parsing/accountsParser.go

```diff
@@ -187,11 +187,11 @@ func (g *Genesis) checkInitialAccount(initialAccount *data.InitialAccount) error
 	return nil
 }
 
-func (g *Genesis) checkForDuplicates() error {
-	for idx1 := 0; idx1 < len(g.initialAccounts); idx1++ {
-		ib1 := g.initialAccounts[idx1]
-		for idx2 := idx1 + 1; idx2 < len(g.initialAccounts); idx2++ {
-			ib2 := g.initialAccounts[idx2]
+func (ap *accountsParser) checkForDuplicates() error {
+	for idx1 := 0; idx1 < len(ap.initialAccounts); idx1++ {
+		ib1 := ap.initialAccounts[idx1]
```

Comment:
maybe ia (initial account)  instead of ib

Reply:
renamed